ci/build_container/build_recipes/boringssl.sh

```diff
@@ -0,0 +1,8 @@
+git clone https://boringssl.googlesource.com/boringssl
+cd boringssl
+git reset --hard b87c80300647c2c0311c1489a104470e099f1531
+cmake .
+make
```

Comment:
By prefixing the shell invocation line with +, the top-level make puts the make coordination flags in the environment, so no need to do anything special in the script.